Example/Source/Examples/Images/ImagesInCollectionBrickViewController.swift

```diff
@@ -33,7 +33,7 @@ class ImagesInCollectionBrickViewController: BrickViewController {
         collectionSection.repeatCountDataSource = self
 
         let section = BrickSection(bricks: [
-            CollectionBrick(dataSource: CollectionBrickCellModel(section: collectionSection, configureHandler: { (cell) in
+            CollectionBrick(dataSource: CollectionBrickCellModel(section: collectionSection, registerBricksHandler: { (cell) in
                 cell.brickCollectionView.registerBrickClass(ImageBrick.self)
```

Comment:
Could you make an example when to use `registerBricksHandler` (basically when you want to register a certain Nib instead of the default implementation)